app/views/user_mailer/welcome_email.html.erb

```diff
@@ -1,274 +1,316 @@
-    </div>
+<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
+<html xmlns="http://www.w3.org/1999/xhtml">
 
-    <div class="login-link">
-      <a href="http://theodinproject.com/login">Login</a>
-    </div>
-  </div>
+<head>
+  <meta http-equiv="Content-Type" content="text/html; charset=utf-8">
 
-  <div class="top-box">
-    <h1>
-      Getting Started with The Odin Project
-    </h1>
-  </div>
+  <!--[if !mso]><!-->
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <!--<![endif]-->
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title></title>
 
-  <div class="content-container">
-    <div class="welcome-header">
-      Welcome to The Odin Project!
-    </div>
+  <!--[if (gte mso 9)|(IE)]>
+    <style type="text/css">
+        table {border-collapse: collapse;}
+    </style>
+    <![endif]-->
+</head>
 
-    <div class="welcome-msg">
-      It's great to have you with us while you learn web development!  It can be a bit overwhelming at first, so to get you started here are a few tips:
-    </div>
+<body style="background-color: #fafafa; margin: 0 !important; padding: 0;">
+  <center class="wrapper" style="-ms-text-size-adjust: 100%; -webkit-text-size-adjust: 100%; table-layout: fixed; width: 100%;">
+    <div class="webkit" style="margin: 0 auto 20px auto; max-width: 600px;">
+      <!--[if (gte mso 9)|(IE)]>
+<table width="600" align="center">
+<tr>
+<td>
+<![endif]-->
 
-    <div class="tip-wrapper">
-      <div class="tip">
-        <div class="tip-number">
-          1
-        </div>
-        <div class="tip-content">
-          <h2>
-            Plan out your strategy
-          </h2>
-          <p>
-            Figure out how much time each week you'll be able to devote to learning then browse through our <%= link_to "roadmap of courses", curriculum_url %> and plan your strategy accordingly.  The biggest issue students have is getting distracted (especially since we link to so much juicy content), so staying focused is important.
-          </p>
-        </div>
-      </div>
+      <table class="padding-table" align="center" style="background-color: #fafafa; border-spacing: 0; color: #333333; font-family: sans-serif; margin: 0; max-width: 600px; vertical-align: bottom; width: 100%;">
+        <tr>
+          <td class="login-row" style="background-color: #fafafa; padding: 0; text-align: right;">
+            <a href="http://theodinproject.com/login" class="login" style="padding: 0;">Login</a>
+          </td>
+        </tr>
+      </table>
+      <table class="outer" align="center" style="background-color: #ffffff; border: 1px solid black; border-spacing: 0; color: #333333; font-family: sans-serif; max-width: 600px; width: 100%;">
+        <tr>
+          <td class="full-width-image center" style="padding: 0; text-align: center;">
+            <%=  image_tag attachments['home-isometric.svg'].url, alt: 'home page banner' %>
```

Comment:
I've just been testing this locally and I'm getting an error here. Is this image actually used anywhere in the email? It's not displaying it anywhere for me.